### PR TITLE
solution

### PR DESCRIPTION
**Pull request by patryk177m**
[DEMO LINK](https://patryk177m.github.io/react_movies-list/)

### CODE REVIEW

**Review by PawelP2001 — CHANGES_REQUESTED** [review]
Looks fine to me, but i think we were also supposed to move .movies and .card blocks from App.scss to MoviesList and MovieCard .scss's (i don't see any changes in those files here, maybe you forgot to commit them?)

**Review by PawelP2001 — APPROVED** [review]

File: src/components/MoviesList/MoviesList.tsx [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+
+export const MoviesList: React.FC<Props> = ({ movies }: Props) => {
+  return (
+    <>
+      <div className="movies">
+        {movies.map((movie: Movie) => {
+          return (
+            <MovieCard movie={{ ...movie }} key={movie.imdbId} />
+          );
+        })}
```
Comment (choeqq) [inline src/components/MoviesList/MoviesList.tsx:18]:
```suggestion
        {movies.map((movie: Movie) => (
            <MovieCard movie={{ ...movie }} key={movie.imdbId} />
        ))}
```

**Review by choeqq — CHANGES_REQUESTED** [review]
Please make sure each component has it's own styles file

**Comment by patryk177m** [comment]
[DEMO LINK](https://patryk177m.github.io/react_movies-list/)

**Comment by patryk177m** [comment]
[DEMO LINK](https://github.com/mate-academy/react_movies-list/pull/1862)

**Comment by patryk177m** [comment]
https://github.com/mate-academy/react_movies-list/pull/1862

**Pull request closed without merging**